bin/test-workflow/start

```diff
@@ -121,8 +131,9 @@ export APP_PLATFORM
 
 trap cleanup EXIT
 
+conjur_init="
 source ./0_prep_env.sh
-./1_deploy_conjur.sh
+./1_deploy_conjur.sh"
```

Comment:
This is outside of our change set, but can you add blank lines at lines 140 and 141 to visually separate the definitions for `conjur_prep`, `cluster_prep`, and `test_app_workflow` strings?

Reply:
done